flink-python/pyflink/table/table_environment.py

```diff
@@ -1154,26 +1154,28 @@ def connect(self, connector_descriptor):
             self._j_tenv.connect(connector_descriptor._j_connector_descriptor))
 
     @staticmethod
-    def create(stream_execution_environment, table_config=None, environment_settings=None):
+    def create(stream_execution_environment=None, table_config=None, environment_settings=None):
         """
-        Creates a :class:`~pyflink.table.TableEnvironment` for a
+        Creates a :class:`~pyflink.table.StreamTableEnvironment` for a
```

Comment:
Change to
```
Creates a :class:`~pyflink.table.StreamTableEnvironment`.
```
since we can also create the TableEnvironment without the execution environment.